app/views/recipes/show.html.haml

```diff
@@ -1,4 +1,6 @@
 %h3= @recipe.title
+%p= @recipe.rating
```

Comment:
Suggestion:
%p= "Rating: #{@recipe.average_rating}" / 5 by #{@recipe.ratings.count} users."

Reply:
Yes this will be added or something similar.